beacon-chain/p2p/fork.go

```diff
@@ -0,0 +1,127 @@
+package p2p
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/ethereum/go-ethereum/p2p/enr"
+	"github.com/prysmaticlabs/go-ssz"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/sirupsen/logrus"
+)
+
+// ENR key used for eth2-related fork data.
+const eth2EnrKey = "eth2"
```

Comment:
Please capitalize all acronyms.

```suggestion
const eth2ENRKey = "eth2"
```

Reply:
+1